src/main/kotlin/org/rust/lang/core/resolve/ItemResolution.kt

```diff
@@ -140,16 +140,6 @@ fun processItemDeclarations(
     return false
 }
 
-private val RsUseSpeck.nameInScope: String? get() {
-    alias?.name?.let { return it }
-    val baseName = path?.referenceName ?: return null
-    if (baseName == "self") {
-        NameResolutionTestmarks.selfInGroupName.hit()
```

Comment:
Unfortunately, I don't know good way to make this code public, not to remove this testmark and not to add some artificial code at the same time
Any idea?

Reply:
I think this is fine :)